src/ontology/oba.Makefile

```diff
@@ -81,12 +81,15 @@ $(MIRRORDIR)/lipidmaps.owl: $(TEMPLATEDIR)/lipidmaps.tsv
 $(ONT)-full.owl: $(SRC) $(OTHER_SRC)
 	echo "INFO: Running FULL release, which is customised for OBA."
 	$(ROBOT) merge --input $< \
-		merge -i components/reasoner_axioms.owl \
```

Comment:
This is removing the following axiom:

```
ReflexiveObjectProperty(<http://purl.obolibrary.org/obo/BFO_0000050>)
```

Due to the global properties of this statement, we are dropping it.